nx/include/switch/services/set.h

```diff
@@ -319,16 +327,17 @@ typedef struct {
 } SetCalConsoleSixAxisSensorHorizontalOffset;
 
 typedef struct {
-    u32 code[0x4];          ///< Country code.
-} SetCalCountryCode;
+    char code[0x3];          ///< Country code.
+} PACKED SetCalCountryCode;
```

Comment:
This PACKED doesn't serve any purpose from what I can tell.

Reply:
```c
    cal_ptr = out_cal_buffer;
    val_ptr = &out_cal_buffer->WlanCountryCodesNum;
    if ( cal_ptr->WlanCountryCodes_checksum == Crc16(&out_cal_buffer->WlanCountryCodesNum, 0x18ELL) )
    {
      code_count = *val_ptr;
      if ( code_count >= code_list_count )
        code_count = code_list_count;
      *out_count = code_count;
      if ( code_count )
      {
        _memmove_device(out_code_list, cal_ptr->WlanCountryCodes, 3LL * code_count);
        LODWORD(code_count) = 0;
      }
    }
    else
    {
      LODWORD(code_count) = 0x48C69;
    }```
From `settings` 9.0.0. It copies exactly `3 * count` bytes from `cal` to the out buffer. It is odd, but I wasn't sure how else to guarantee that, or whether this even does, but surely you'd get bad indices if we don't guarantee a 3-byte struct size. My `cal` only has one entry, as do many I imagine, so I'm not sure how to test this.

Reply:
This structure contains an array of 3 u8s. Alignment of u8 is 1. I don't think anything is changed, or being forced to 4 byte alignment without the PACKED.

Reply:
you're right, i tested this in homebrew:
```c
typedef struct {
    char code[0x3];          ///< Country code.
} CountryCodeTest;

CountryCodeTest cctest[2];
printf("size of CountryCodeTest * 2 == %x\n", sizeof(cctest));
```
yields `size of CountryCodeTest * 2 == 6`